src/Symfony/Component/VarDumper/Caster/MemcachedCaster.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class MemcachedCaster
+{
+    public static $optionConstants;
+    public static $defaultOptions;
+
+    public static function castMemcached(\Memcached $c, array $a, Stub $stub, $isNested)
+    {
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'servers' => $c->getServerList(),
+            Caster::PREFIX_VIRTUAL.'options' => new EnumStub(
+                self::getMemcachedNonDefaultValueOptions($c)
+            ),
+        );
+
+        return $a;
+    }
+
+    public static function getMemcachedNonDefaultValueOptions(\Memcached $c)
```

Comment:
should be private !
I'd suggest renaming meanwhile: `public static function getNonDefaultOptions`